src/main/java/com/xpanxion/java/springboot/da1/demo/service/student1/GymService1.java

```diff
@@ -25,4 +32,18 @@ public void addGym(Gym1 gym) {
 
     }
 
+    public AtomicReference<String> addGymMember(int gymId, GymMember1 gymMember1) {
+
+        AtomicReference<String> result = new AtomicReference<>("Gym Added!");
```

Comment:
I'm not entirely sure this is a proper user of the `AtomicReference`, but I like the effort so let's keep it.  :-)

Reply:
haha Thanks! I'll do more research